packages/backend/src/core/collectors/UserTransactionCollector.ts

```diff
@@ -70,20 +89,8 @@ export class UserTransactionCollector {
       timestamp: Timestamp.fromSeconds(timestamp),
     }
 
+    let record
```

Comment:
We could type this here

Reply:
Given how simple update this is, I'll do it in a separate PR, because there are other PRs that rely on this one and I want to keep merging simple.